healthvault-ios-sdk/Classes/Controllers/MHVBrowserAuthBroker.h

```diff
@@ -18,7 +18,8 @@
 
 #import "MHVBrowserController.h"
 #import "MHVBrowserAuthBrokerProtocol.h"
+#import <WebKit/WebKit.h>
 
-@interface MHVBrowserAuthBroker : MHVBrowserController<MHVBrowserAuthBrokerProtocol>
+@interface MHVBrowserAuthBroker : NSObject <MHVBrowserAuthBrokerProtocol>
```

Comment:
We should probably move this out of the Controllers directory and put it in the Authentication
 directory.